scripts/count_monty_params.py

```diff
@@ -0,0 +1,165 @@
+# Copyright 2025 Thousand Brains Project
+#
+# Copyright may exist in Contributors' modifications
+# and/or contributions to the work.
+#
+# Use of this source code is governed by the MIT
+# license that can be found in the LICENSE file or at
+# https://opensource.org/licenses/MIT.
+
+"""Count the number of parameters in the `dist_agent_1lm`  Monty model.
+
+This script counts the number of parameters in a Monty model, specifically the
+`dist_agent_1lm` pretrained model. This produces the result (4 million parameters)
+referenced in the Continual Learning section of the paper.
+
+The Monty model consists of a series of learned graphs for different objects. These
+are stored in a nested state dictionary, e.g.:
+
+    graph = state_dict["lm_dict"][lm_id]["graph_memory"]["mug"]["patch"]
+
+This script will count all the numerical values stored in these graphs, as a measure
+of the total number of parameters in the model.
+
+For clarity, we begin by breaking down the structure of the state dictionary.
+
+A Monty model's state_dict contains the following keys:
+- 'lm_dict'
+- 'sm_dict'
+- 'motor_system_dict'
+- 'lm_to_lm_matrix'
+- 'lm_to_lm_vote_matrix'
+- 'sm_to_lm_matrix'
+
+All of the information we are concerned with is in lm_dict. sm_dict and
+motor_system_dict contain information for logging and visualization purposes. The
+connectivity information in lm_to_lm_matrix, lm_to_lm_vote_matrix, and sm_to_lm_matrix
+is a negligible fraction of data, particularly in single-LM models.
+
+Within lm_dict, lm_id is the index of the Learning Module(s), of which only 0 is
+relevant for single-LM models. lm_dict does not contain any other keys.
+
+state_dict["lm_dict"][lm_id] contains:
+- 'graph_memory'
+- 'target_to_graph_id'
+- 'graph_id_to_target'
+The latter two are mappings between the object name and the index of the graph, which
+are needed to assess the performance of the model. They are of negligible size.
```

Comment:
```suggestion
are needed to assess the performance of the model. 
```
I don't think you need to mention the size here as they are just used for logging and are not part of the model